README.md

```diff
@@ -197,3 +207,16 @@ Object with the default options for the notifications
 
 ## defaultOptionsByType
 Object with the default options for the notifications for specific types
+
+## Building
+
+_OS X users, ensure you have `wget` installed (available in `brew`)_
+
+To run the tests:
+
+```bash
+$ ./workaround.sh
```

Comment:
@hiddentao Yes thats because you are running the script ment for travis.
I'd recommend you to just use ```meteor test-packages ./```